crisp_api/__init__.py

```diff
@@ -84,7 +84,7 @@ def __do_request(self, method, resource, query=None, data=None):
             headers=self.REQUEST_HEADERS,
             auth=auth,
             params=query,
-            data=(json.dumps(data) if data != None else None)
+            data=(json.dumps(data) if data is not None else None)
```

Comment:
This is right, thanks!

Reply:
I will submit a commit to change to 2-spaces indentation and restore "()"